src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -24,7 +24,8 @@ public static class FakesUtilities
 
         private const string CrossPlatformConfiguratorMethodName = "GetCrossPlatformDataCollectorSettings";
 
-        private const string FakesConfiguratorAssembly = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=16.0.0.0, Culture=neutral";
+        // Must be kept in sync with version being generated by VSUnitTesting
+        private const string FakesConfiguratorAssembly = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=17.0.0.0, Culture=neutral";
```

Comment:
consider: There seems to be a lot of places in the repo where the string `16.0.0.0` is used. I think it would be great to pull this from a central .cs file that can be updated for every release. Since it's just a .cs file, the consumers wouldn't have an additional dll reference.
/cc: @cvpoienaru

Reply:
Would this be something I add here? It sounds like it would have to be a repo wide change that I wouldn't want to make. We could adopt to a standard if it's set in vstest. For now I've shared the versions in this file.
@cvpoienaru do you have recommendations on whether I could use a central file to pull the version string from?

Reply:
yes, I was recommending a repo wide file. We could put it in the eng folder where version updates are usually made.

Reply:
The versions are located in 2 places. in the eng folder exists version for packages, and in scripts\build folder exists versions for dependencies. The Fakes version currently exists in scripts\build, and so I added a central .cs file there

Reply:
sounds good

Reply:
Thanks for taking the time to make the changes Abhitej proposed, @vritant24. Indeed I think the `scripts\build` folder works better overall for placing the new `.cs` file.

On a side note I see the PR has a merge conflict so the build hasn't run yet. Does this change build locally ? I'm asking because it's not completely clear to me how the `.cs` file ends up being pulled in from `scripts\build` & compiled with the project in case, i.e `Microsoft.TestPlatform.Common.csproj`. Should we add a compile directive for that file in the `.csproj` ?

Reply:
Looks like I forgot to commit the csproj file. added.
Thanks for pointing that out!